src/components/sidebar/CallTreeSidebar.js

```diff
@@ -102,25 +97,73 @@ class CategoryBreakdown extends React.PureComponent<CategoryBreakdownProps> {
   render() {
     const { breakdown, categoryList, isIntervalInteger } = this.props;
     const data = breakdown
-      .map((value, categoryIndex) => {
+      .map((oneCategoryBreakdown, categoryIndex) => {
         const category = categoryList[categoryIndex];
         return {
-          group: category.name,
-          value: value || 0,
-          color: category.color,
+          category,
+          value: oneCategoryBreakdown.entireCategoryValue || 0,
+          subcategories: category.subcategories
+            .map((subcategoryName, subcategoryIndex) => ({
+              name: subcategoryName,
+              value:
+                oneCategoryBreakdown.subcategoryBreakdown[subcategoryIndex],
+            }))
+            // sort subcategories in descending order
+            .sort(({ value: valueA }, { value: valueB }) => valueB - valueA)
+            .filter(({ value }) => value),
         };
       })
-      // sort in descending order
-      .sort(({ value: valueA }, { value: valueB }) => valueB - valueA);
+      // sort categories in descending order
+      .sort(({ value: valueA }, { value: valueB }) => valueB - valueA)
+      .filter(({ value }) => value);
 
-    return <Breakdown data={data} isIntervalInteger={isIntervalInteger} />;
+    const totalTime = data.reduce((accum, { value }) => accum + value, 0);
+    const maxFractionalDigits = isIntervalInteger ? 0 : 1;
+
+    return (
+      <div className="sidebar-categorylist">
+        {data.map(({ category, value, subcategories }) => {
+          const percentage = Math.round((value / totalTime) * 100);
+          return (
+            <React.Fragment key={category.name}>
+              <div className="sidebar-categoryname">
+                <span
+                  className={`sidebar-color colored-square category-color-${
+                    category.color
+                  }`}
+                  title={category.name}
+                />
+                {category.name}
+              </div>
+              <div className="sidebar-categorytiming">
+                {formatMilliseconds(value, 3, maxFractionalDigits)} (
+                {percentage}%)
```

Comment:
We have a `formatPercent` utility in `format-numbers` too, that do it in a locale-friendly way: `formatPercent(value / totalTime)`

(same remark for the other use below)


(and I see we didn't use it in the existing location, so this is a non-blocking comment)